manifests/kustomize/env/aws/config

```diff
@@ -1,6 +1,5 @@
 {
 namespace: $(kfp-namespace),
-executorImage: gcr.io/ml-pipeline/argoexec:v2.7.5-license-compliance,
```

Comment:
Do we need to replace this or how does it work?

Reply:
it's also specified in workflow controller argument.
and it seems that it has been removed from https://github.com/argoproj/argo-workflows/blob/v2.12.9/docs/workflow-controller-configmap.yaml